Produlytics/ui/front-end/src/app/model/chart/chart.service.ts

```diff
@@ -43,8 +43,8 @@ export class ChartService implements ChartAbstractService{
    */
   getNextPoints(deviceId: number, characteristicId: number, newerThan: number): Observable<ChartPointReturn> {
     const paramsObj = {
-      newerThan: newerThan,
-      limit: 1
+      newerThan: newerThan-13000,
+      limit: 14
```

Comment:
Cosa succede se il back-end ha più di 14 punti da inviare?

Reply:
Se guardi ora ho messo `limit: 16`, l'idea è che prendo 15 rilevazioni che già ho per aggiornarle più una "nuova" da aggiungere. Si potrebbe incrementare, ma non  saprei che limite porre. Inoltre nei requisiti abbiamo scritto che dobbiamo sopportare _almeno_ 1 rilevazione al secondo, quindi in questo modo viene soddisfatto.